packages/api/src/router/user.ts

```diff
@@ -420,12 +422,12 @@ export const userRouter = createTRPCRouter({
         })
         .where(eq(users.id, input.userId));
     }),
-  changeDefaultSearchEngine: protectedProcedure
+  changeSearchPreferences: protectedProcedure
     .input(
-      convertIntersectionToZodObject(validation.user.changeDefaultSearchEngine.and(z.object({ userId: z.string() }))),
+      convertIntersectionToZodObject(validation.user.changeSearchPreferences.and(z.object({ userId: z.string() }))),
     )
     .output(z.void())
-    .meta({ openapi: { method: "PATCH", path: "/api/users/changeSearchEngine", tags: ["users"], protect: true } })
+    .meta({ openapi: { method: "PATCH", path: "/api/users/search-preferences", tags: ["users"], protect: true } })
```

Comment:
This would be a breaking change...
So either mark the commit as breaking, revert this change or add a backwards compatible and deprecated marked endpoint.

Reply:
Created a task to remove it once we release 2.0:
https://github.com/homarr-labs/homarr/issues/2136